testplan/runnable/interactive/base.py

```diff
@@ -139,17 +140,13 @@ def target(self):
         """
         return self._cfg.target
 
-    def abort_dependencies(self):
-        """
-        We abort our test runner instance.
-        """
-        yield self.target
-
     def aborting(self):
```

Comment:
Move self._http_handler.stop() to teardown and call teardown from aborting?

Reply:
As mentioned above this function is removed but add `abort_dependencies`